docs/rules/consistent-function-scoping.md

```diff
@@ -0,0 +1,44 @@
+# Prefer consistent function scoping
```

Comment:
Can you come up with something more descriptive that doesn't just repeat the rule name?